django_react_proj/users-fe/package.json

```diff
@@ -20,7 +20,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "eslint": "eslint ./src",
+    "fix": "eslint --fix ./src .eslintrc.js",
+    "format": "prettier --write ./src"
```

Comment:
If you have integration with prettier and eslint `eslint --fix` does the same as this